spec/models/ems_refresh/refreshers/openstack/refresh_spec_common.rb

```diff
@@ -163,11 +181,15 @@ def assert_specific_flavor
                                  [:is_public, :disk, :ephemeral, :swap]) # TODO(lsmola) model blacklisted attrs
 
       ManageIQ::Providers::Openstack::CloudManager::Flavor.all.each do |flavor|
+        disk_size, disk_count = name_to_disk_info(flavor.name)
+
         expect(flavor.ext_management_system).to eq @ems
         # TODO(lsmola) expose below to Builder's data
         expect(flavor.enabled).to               eq true
         expect(flavor.cpu_cores).to             eq nil
         expect(flavor.description).to           eq nil
+        expect(flavor.disk_size).to             eq disk_size
+        expect(flavor.disk_count).to            eq disk_count
```

Comment:
To properly test the algorithm I think it could even be cleaner with:

`expect(flavor.disk_count).to            eq flavor.disk_size == 0 ? 0 : 1`